spswarehouse/general/selenium.py

```diff
@@ -1,53 +1,62 @@
+import time
+
 from selenium.webdriver.support.ui import WebDriverWait, Select
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
 
 
+### Internal Functions
+
```

Comment:
My vision for this particular file was that any tool could import any of these functions and use them anywhere. But I agree that a general SPSSelenium class would be even more useful, and that could just be our default for using Selenium across our tool. Since that's a decent change in scope, I just create an Asana task to put on the backlog [here](https://app.asana.com/0/1203184926617908/1205543700128559).